### PR TITLE
Camera editor sensitivity changes fixes #133

### LINKED ISSUE #133: Improving editor camera sensitivity
**Problem this feature should fix**
After using a lot of other popular game engines (Unity, Unreal Engine), going back to Overload is always weird because the camera movements (Right click and middle click camera movements) are kind of off (Uncommon sensitivity).

**Expected solution**
We should try inspiring our camera rotation and translation speeds (Right click and middle click) from other game engines, such as Unity.

### PR DESCRIPTION
**Pull request by sgavil**
I have changed camera sensitivity, speed and drag with inspiration from Unity. In order to get that similiar feeling I have increased those values a bit.

### CODE REVIEW

File: Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h [hunk 1]
```diff
@@ -124,10 +124,10 @@ namespace OvEditor::Core
 		double m_lastMousePosX = 0.0;
 		double m_lastMousePosY = 0.0;
 		OvMaths::FVector3 m_ypr;
-		float m_mouseSensitivity = 0.05f;
-		float m_cameraDragSpeed = 0.01f;
+		float m_mouseSensitivity = 0.12f;
+		float m_cameraDragSpeed = 0.07f; 
 		float m_cameraOrbitSpeed = 0.5f;
-		float m_cameraMoveSpeed = 5.0f;
+		float m_cameraMoveSpeed = 8.0f;
```
Comment (adriengivry) [inline Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h:130]: This setting shouldn't get changed as it is exposed in the UI (Required when working with environments with different scales)

File: Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h [hunk 2]
```diff
@@ -124,10 +124,10 @@ namespace OvEditor::Core
 		double m_lastMousePosX = 0.0;
 		double m_lastMousePosY = 0.0;
 		OvMaths::FVector3 m_ypr;
-		float m_mouseSensitivity = 0.05f;
-		float m_cameraDragSpeed = 0.01f;
+		float m_mouseSensitivity = 0.12f;
+		float m_cameraDragSpeed = 0.07f;
```
Comment (adriengivry) [inline Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h:128]: This one is a bit too much in my opinion. I've tested it and `0.03f` looked more appropriate. I know that this is totally subjective tho

Comment (sgavil) [reply Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h:128]: I had Unity open in order to compare the feeling. Maybe it was too much yes, but my general feeling was that the movement felt quite heavy

Comment (adriengivry) [reply Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h]: I was actually only talking about the `m_cameraDragSpeed`, the `m_mouseSensitivity` of `0.12f` felt good IMO

Comment (sgavil) [reply Sources/Overload/OvEditor/include/OvEditor/Core/CameraController.h]: Okey! then `m_cameraDragSpeed` is set to `0.12` and `m_mouseSensitivity` to `0.03` I think is much better like this

**Comment by adriengivry** [comment]
All good for camera sensitivity changes. However, the PR is becoming a bit messy, I guess you merged develop onto the PR branch or something. It would be nice if you can only isole your change in this PR.
I suggest that you use rebase when you want to keep your branch up-to-date with the master repository.

**Comment by sgavil** [comment]
@adriengivry I rebased the 9 commits that came with the merge from develop. If this is not okay I would isolate the changes in another PR as you suggest. Thanks

**Comment by adriengivry** [comment]
@sgavil Yeah, I think it would be good to create another PR. The problem is that even after a rebase, your fork history has diverged with the origin

**Pull request closed without merging**